antora.yml

```
@@ -1,6 +1,6 @@
 name: desktop
 title: Desktop App
-version: 'next'
+version: '5.1'
```

```suggestion
version: 'next'
```
Reverting. Do not fiddle on antora build stuff. If you have questions you are welcomed to ask.